Code/tests/test_filefinder.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+import plistlib
+import StringIO
+import unittest
+
+import mock
+from autopkglib import ProcessorError
+from autopkglib.FileFinder import FileFinder
+
+
+class TestFileFinder(unittest.TestCase):
+    """Test class for FileFinder Processor."""
+
+    def setUp(self):
+        self.good_env = {"find_method": "glob", "pattern": "test"}
+        self.bad_env = {"find_method": "fake"}
+        self.input_plist = StringIO.StringIO(plistlib.writePlistToString(self.good_env))
+        self.processor = FileFinder(infile=self.input_plist)
```

Comment:
This is the general framework for testing a Processor - you add it as a class attribute in the setup, and then you can use it whenever you call main().